pandas/tests/arithmetic/test_interval.py

```diff
@@ -271,3 +271,18 @@ def test_index_series_compat(self, op, constructor, expected_type, assert_func):
         result = op(index, other)
         expected = expected_type(self.elementwise_comparison(op, index, other))
         assert_func(result, expected)
+
+
+@pytest.mark.parametrize("method", ["__add__", "__sub__"])
+def test_timestamp_interval_add_subtract_timedelta(method):
+    # https://github.com/pandas-dev/pandas/issues/32023
+    interval = Interval(
+        Timestamp("2017-01-01 00:00:00"), Timestamp("2018-01-01 00:00:00")
```

Comment:
since this is testing the scalar, this test belongs in tests.scalar.interval, could start a new file test_arithmetic.py there (looks like there are some `test_math_foo` tests in test_interval.py)

this module would be for testing the behavior of IntervalArray, IntervalIndex, Series[IntervalDtype], DataFrame[IntervalColumn]

Reply:
Updated the tests but haven't gotten around to making them pass them, will try to fix these later